README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+---
+output:
+  pdf_document: default
+  word_document: default
+  html_document: default
+---
+# Caleb Weis
+## Computing for the Social Sciences
+### Homework 1
+
+![Profile Picture](IMG_7332.JPG)
```

Comment:
I couldn't view the picture properly. I am not sure whether you have uploaded the picture to the Github. Maybe you want to double-check that